test/test-async.js

```diff
@@ -4267,5 +4267,49 @@ exports['asyncify'] = {
             test.ok(e.message === "callback error");
             test.done();
         }
-    }
+    },
+
+    'promisified': [
+        'native-promise-only',
+        'bluebird',
+        'es6-promise',
+        'rsvp'
+    ].reduce(function(promises, name) {
+        var Promise = require(name);
```

Comment:
You might want to make these tests node-only, because this won't work in the browser.